src/components/guess-form.js

```diff
@@ -2,21 +2,51 @@ import React from 'react';
 
 import './guess-form.css';
 
-export default function GuessForm(props) {
-    //console.log(props);
-    function handleSubmit(e) {
-        e.preventDefault();
-        props.increment();
-        props.updateCurrentGuess(Number(e.target.userGuess.value))
-        e.target.userGuess.value = '';
+class GuessForm extends React.Component {
```

Comment:
This is for controlled my input. This is what we call a controlled input

https://www.youtube.com/watch?v=nSGZEQa5C_c